macos_build.sh

```diff
@@ -4,8 +4,9 @@
 set -e
 
 rsync -ah x64-osx-openrct2/* universal-osx-openrct2
-for lib in x64-osx-openrct2/lib/*.dylib; do
+find x64-osx-openrct2/lib -type f -name "*.dylib" | while IFS= read -r lib; do
```

Comment:
What's the point of the `IFS` variable here? Couldn't we just do:
```suggestion
find x64-osx-openrct2/lib -type f -name "*.dylib" | while read -r lib; do
```

Reply:
> What's the point of the `IFS` variable here? 
IFS is a special variable in bash, setting it to an empty value means the read command will treat the entire line as a single value without splitting it on whitespace. It might not be strictly needed but probably good to have when looping over a find.